### DISCUSSION TITLE
[@types/node]: Readable Streams Does NOT Have open & ready Event

### DISCUSSION THREAD

**Discussion by mahdi-farnia** [DefinitelyTyped/DefinitelyTyped#61469] [Issues with a @types package] [answered]
Nodejs readable streams (i did not test the others) does not have `open` & `ready` events.
Also according to docs, there is no such an event with these names...

**Link to docs:** https://nodejs.org/api/stream.html
**Package Version:** 18.6.1

## Edit

**I have tested the events and they are working properly, may be those events have missed written to the docs**

[a comment by an automated account is not shown]

**Comment by mcollina** [comment, marked as answer]
They don't because those events are not part of the standard Node.js Stream interface. Those are _a convention_ that is specific of fs.createReadStream() and others.

**Reply by mahdi-farnia** [reply]
thanks 🙏
